src/services/MemberService.java

```diff
@@ -75,13 +84,50 @@ public void update(Member obj) throws SQLException {
 
     @Override
     public Member get(Member obj) throws SQLException {
-	throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+
+        String condition = "";
+        if (obj.getId() != 0) {
+            condition = "Where id = " + obj.getId();
+        } else if (obj.getPseudo() != null) {
+            condition = "Where pseudo = " + obj.getPseudo();
+        } else if (obj.getEmail() != null) {
+            condition = "Where email = " + obj.getEmail();
+        }
+        String req = "Select * , date_format( birth_date ,'%d %m %Y') as birth from user " + condition;
+        st = CONNECTION.createStatement();
+        rs = st.executeQuery(req);
+        rs.next();
+        obj.setPseudo(rs.getString("pseudo"));
+        obj.setFirstname(rs.getString("firstname"));
+        obj.setLastname(rs.getString("lastname"));
+        obj.setEmail(rs.getString("email"));
+        obj.setPassword(rs.getString("password"));
+        obj.setBirthDate(rs.getDate("birth_date"));
+        obj.setGender(rs.getBoolean("gender"));
+        obj.setHeight(rs.getFloat("height"));
+        obj.setBodyType(Enumerations.BodyType.values()[rs.getInt("body_type")]);
+        obj.setChildrenNumber(rs.getInt("children_number"));
+        obj.setReligion(Enumerations.Religion.values()[rs.getInt("relegion")]);
+        obj.setReligionImportance(Enumerations.Importance.values()[rs.getInt("relegion_importance")]);
+        obj.setSmoker(rs.getBoolean("smoker"));
+        obj.setDrinker(rs.getBoolean("drinker"));
+        obj.setMinAge(rs.getInt("min_age"));
+        obj.setMaxAge(rs.getInt("max_age"));
+        obj.setProximity(Enumerations.Proximity.values()[rs.getInt("proximity")]);
+        obj.setLastLogin(rs.getTimestamp("last_login"));
+        obj.setLocked(rs.getShort("locked"));
+        obj.setIp(rs.getString("ip"));
+        obj.setPort(rs.getInt("port"));
+        obj.setAddress(new Address(obj.getId()));
```

Comment:
getAddress from addressService
obj.setAddress(AddressService.getInstance().get(new Address(obj.getId()));

Reply:
you forgot to set the id as well.

Reply:
@elyes007 which id? ken u're talking about user_id, donc there is no need for it, since it's already set